src/backend/catalog/catalog.c

```diff
@@ -52,6 +52,7 @@
 IsExtendedCatalogHookType IsExtendedCatalogHook;
 IsToastRelationHookType IsToastRelationHook;
 IsToastClassHookType IsToastClassHook;
+IsTdsClientHookType IsTdsClientHook;
```

Comment:
We cannot use TDS keyword in Opensource Engine, please follow guidelines before raising such changes